asmcli/commands/create-mesh.sh

```diff
@@ -33,6 +34,10 @@ create-mesh_parse_args() {
         context_set-option "PRINT_VERSION" 1
         shift 1
         ;;
+      --no_auto_trusting_fleet_identity | --no-auto-trusting-fleet-identity)
```

Comment:
Let's use something like `--ignore_workload_identity_mismatch`. Skipping this might break customers, and if it just says "don't trust" it's not obvious that *not* doing this is dangerous.